### ISSUE TITLE
Map not showing Pokemon

### ISSUE THREAD

**Issue by eppcmogen** [AHAAAAAAA/PokemonGo-Map#3274]
Recently, my console has been giving me a lot of "Upserted 0 pokemon" and occcasionally "20-50 poketstops", etc. I think Niantic changed something in the backend side of things to that's causing the scanner to miss the pokemon because a week ago, before the throttling and the update, it was working perfectly. Anybody else having this issue?

![image](https://cloud.githubusercontent.com/assets/13462253/17355022/9376d0be-591b-11e6-9cba-e3eef07b0d27.png)

**Comment by freakjam** [comment]
-sd 10

**Comment by eppcmogen** [comment]
@freakjam that's going to significantly slow down the scanning (-scan delay), is there a sweet spot? Would keeping it at 3 requests per second optimize the scanner?

**Comment by skeldoor** [comment]
Max is one request per 10 seconds

**Comment by eppcmogen** [comment]
@skeldoor ah oh geez. Well it was fun while it lasted. Btw, are your Pokemon showing up? It's upserting pokemon but its not being displayed on the web map

**Comment by zak66601** [comment]
80% of all pokemon i see are from lures now. is runnin -t 5 on 2.1 bad?

**Comment by Nickster65** [comment]
@skeldoor Where did you hear this? Cause if that's true then this isn't really practical anymore...

**Comment by MrKetchumAll** [comment]
@zak66601 Yeah bruh, I got the same. What's up with that lure pokemon only? Hit me up if you find anything.

**Comment by MrKetchumAll** [comment]
I don't know where you guys hearing about this -sd 10, but I get same results using 10 or 5, with 5 being faster.

**Comment by eagle132** [comment]
I am having the same issue. It's just not finding anything even though I know they are there. It will occasionally find one or two but in general...

`python.exe runserver.py -u REDACTED -p REDACTED -st 10 -k REDACTED -l "42.406365 -71.077494" -st 5 -H 0.0.0.0 -L en -t 1 -sd 5 -ld 1 -td 5"`

`2016-08-03 02:28:47,785 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:28:53,098 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:28:58,410 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:29:03,759 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:29:09,099 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:29:14,433 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:29:19,779 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:29:25,217 [        models] [   INFO] Upserted 0 pokemon, 2 pokestops, and 0 gyms
2016-08-03 02:29:30,526 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:29:35,852 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:29:41,444 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:29:46,769 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:29:52,371 [        models] [   INFO] Upserted 0 pokemon, 11 pokestops, and 2 gyms
2016-08-03 02:29:57,713 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:30:03,026 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:30:08,349 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:30:14,042 [        models] [   INFO] Upserted 0 pokemon, 36 pokestops, and 5 gyms
2016-08-03 02:30:19,811 [        models] [   INFO] Upserted 3 pokemon, 36 pokestops, and 5 gyms
2016-08-03 02:30:25,125 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:30:30,480 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:30:36,157 [        models] [   INFO] Upserted 0 pokemon, 35 pokestops, and 4 gyms
2016-08-03 02:30:41,490 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:30:46,858 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:30:52,164 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:30:57,493 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:31:03,072 [        models] [   INFO] Upserted 1 pokemon, 2 pokestops, and 0 gyms
2016-08-03 02:31:08,398 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:31:13,698 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 02:31:19,040 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms`

This is in a major city and in this radius should be tens of pokemon :(

**Comment by YuriMB** [comment]
Maybe use a list of accounts to speed up the scanning process.

**Comment by zak66601** [comment]
I am using 5 acc but 80% of all pokemon i see are from lures now. Maybe even more

**Comment by YuriMB** [comment]
Are you using the latest develop? Seeing 141 here right now.

**Comment by zak66601** [comment]
2.1 on windows

**Comment by MrKetchumAll** [comment]
@YuriMB Are you using Master or Develop?

**Comment by zak66601** [comment]
i got it from https://jonaharagon.github.io/PoGoMapWiki/#!index.md 2.1
Not sure

**Comment by YuriMB** [comment]
Develop

**Comment by btcode** [comment]
#3221

**Comment by eagle132** [comment]
I updated to the develop version and rebuilt and it does seem to be showing more pokemon now. I'm not sure if it's getting all, though that could be due to the api call delay? But it certainly seems better.

**Comment by MrKetchumAll** [comment]
@zak66601 You want to use develop branch. Seems better right now.

**Comment by YuriMB** [comment]
Good!

**Comment by eagle132** [comment]
This is going to seem like an obvious thing, but I wanted to comment on it just in case it was affecting anyone else. Even with the new develop version I was still certain that most of the pokemon were not showing up, and this was correct. The issue ended up being that I had installed Smart Poke V2 (Android App) on my phone and that app does NOT like to close. Even when manually killed it was still running scans in the background using my PTC account that I was using for PokmonGo-Map here. Because of this, there were too many API calls going out and most of the scans were likely failing. 

I force stopped Smart Poke V2 and my scans using PokemongGo-Map have been a whole heck of a lot better.

Once again this seems obvious, but maybe it will help someone :)
